minishift_sanity.py

```diff
@@ -118,10 +120,10 @@ def test_ms_setup_cdk(self):
         self.log.info("Console output: \n" + str(child.before) + str(child.after))
         if not index==0:
             self.fail("CDK setup failed")
-        self.log.info("Checking if rhel.iso exists")
-        exists = os.path.isfile( os.environ['HOME'] + "/.minishift/cache/iso/minishift-rhel.iso" )
+        self.log.info("Checking if minishift-rhel7.iso exists")
+        exists = os.path.isfile( os.environ['HOME'] + "/.minishift/cache/iso/minishift-rhel7.iso" )
```

Comment:
we should not hard code this ```minishift-rhel7.iso```. Can we look for .iso file is present in the dir or not.

Something like 
```
if "CYGWIN" in self.platform:
    if not any(fname.endswith('.iso') for fname in os.listdir(cygpath +"/cache/iso/")):
        self.fail("minishift rhel iso is not present")
    else:
        if not any(fname.endswith('.iso') for fname in os.listdir(os.environ['HOME'] + "/.minishift/cache/iso/")):
            self.fail("minishift rhel iso is not present")
```